### DISCUSSION TITLE
Support animated profile pictures?

### DISCUSSION THREAD

**Discussion by CodedOre** [CodedOre/NewCaw#16] [Backend]
Mastodon allows users to use a GIF as their profile image or header.

We could theoretically support this. `Animated` will likely be using a video in an `Gtk.MediaFile`, which can displayed anywhere where a `Gdk.Paintable` can be used, including `Adw.Avatar`.

Question though is if this is something we want to support (or if anybody is actually using this).

**Comment by IBBoard** [comment]
If Mastodon supports it then we might as well support it 🙂 Just as long as there's a way to use a "normal" avatar widget for static avatars and it doesn't end up using lots of memory when 99% of people have non-animated avatars.

(Although I'd put it as low priority, because the only animated avatars that I've seen are normally bright, garish and annoying, so we can live without it at first 😄)
